pom.xml

```diff
@@ -41,13 +41,12 @@
     
     <properties>
         <wildfly.cekit.modules.fork>wildfly</wildfly.cekit.modules.fork>
-        <wildfly.cekit.modules.tag>0.27.12</wildfly.cekit.modules.tag>
+        <wildfly.cekit.modules.tag>0.27.13</wildfly.cekit.modules.tag>
         <version.junit>4.13.2</version.junit>
-        <!-- version from a test release to unzip in local maven cache -->
-        <version.org.eap>8.0.0.Beta-redhat-20220923</version.org.eap>
-        <version.org.wildfly.core>19.0.0.Beta16-redhat-00002</version.org.wildfly.core>
+        <version.org.eap>8.0.0.Beta-redhat-00003</version.org.eap>
```

Comment:
Can we change that property? `org.eap` looks weird. Could we use instead `version.org.jboss.eap`?